packages/cli/src/commands/source/push/add.ts

```diff
@@ -41,6 +42,13 @@ export default class SourcePushAdd extends Command {
       description:
         'One or multiple folder containing json files. Can be repeated',
     }),
+    maxConcurrent: flags.integer({
+      exclusive: ['file'],
+      char: 'c',
+      default: 10,
+      description:
+        'The maximum number of requests to send concurrently. Increasing this value will increase the speed at which documents are pushed to the Coveo platform. However, if you run into memory or throttling issues, consider reducing this value.',
```

Comment:
```suggestion
        'The maximum number of requests to send concurrently. Increasing this value increases the speed at which documents are pushed to the Coveo platform. However, if you run into memory or throttling issues, consider reducing this value.',
```
Looks good to me 👍 
(I suspect you were speaking to me rather than JPLachance 😛 )

Reply:
oups 🤦  sorry